src/NetAnalyzers/CSharp/GlobalSuppressions.cs

```diff
@@ -0,0 +1,5 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE0130:Namespace does not match folder structure", Justification = "<Pending>", Scope = "namespace", Target = "~N:Microsoft.NetCore.CSharp.Analyzers.Runtime")]
```

Comment:
❗ This repository doesn't use global suppressions. The rule can be configured in the appropriate **.globalconfig** file.

Reply:
> This is a really complicated analysis scenario. I ran into a large number of issues working in this space in the past and I'm not sure a pull request review is the right place to try and cover them all.

Is there a slack or discord channel where we could discuss this further?

Reply:
Fixed.